### ISSUE TITLE
Profiling page stays empty

### ISSUE THREAD

**Issue by rchl** [getsentry/profiling-node#84]
### Is there an existing issue for this?

- [X] I have checked for existing issues https://github.com/getsentry/profiling-node/issues
- [X] I have reviewed the documentation https://docs.sentry.io/
- [X] I am using the latest SDK release https://github.com/getsentry/profiling-node/releases

### How do you use Sentry?

Sentry Saas (sentry.io)

### SDK Version

7.28.0

### What environment is your node script running in?

Standalone script running in Node v16.15.1

### How is your code deployed and bundled?

does not apply

### Steps to Reproduce

1. Create `script.js`:

```js
const axios = require('axios');
const Sentry = require("@sentry/node");
// Note: You MUST import @sentry/tracing package before @sentry/profiling-node
require("@sentry/tracing");
const { ProfilingIntegration } = require("@sentry/profiling-node");

async function test() {
  Sentry.init({
    dsn: '<dsn>',
    environment: 'development',
    debug: true,
    integrations: [
      new ProfilingIntegration()
    ],
    profilesSampleRate: 1.0,
    tracesSampleRate: 1.0,
  });


  const transaction = Sentry.startTransaction({
    op: "transaction",
    name: "My Transaction",
  });

  await axios.default.get('https://github.com/')
  // console.log(response.headers)

  transaction.finish();

  setTimeout(() => {}, 10000)
}

test()
```

2. Run `node ./script.js`

### Expected Result

A new entry shows up on the `Profiling` page.

### Actual Result

Running the script produces this log:

```
Sentry Logger [log]: Integration installed: InboundFilters
Sentry Logger [log]: Integration installed: FunctionToString
Sentry Logger [log]: Integration installed: Console
Sentry Logger [log]: Integration installed: Http
Sentry Logger [log]: Integration installed: OnUncaughtException
Sentry Logger [log]: Integration installed: OnUnhandledRejection
Sentry Logger [log]: Integration installed: ContextLines
Sentry Logger [log]: Integration installed: Context
Sentry Logger [log]: Integration installed: Modules
Sentry Logger [log]: Integration installed: RequestData
Sentry Logger [log]: Integration installed: LinkedErrors
Sentry Logger [log]: Integration installed: ProfilingIntegration
Sentry Logger [log]: [Tracing] starting transaction transaction - My Transaction
Sentry Logger [log]: [Profiling] started profiling transaction: My Transaction
Sentry Logger [log]: [Profiling] stopped profiling of transaction: My Transaction
Sentry Logger [log]: [Tracing] Finishing transaction transaction: My Transaction.
Sentry Logger [log]: [Profiling] Preparing envelope and sending a profiling event.
```

The `Profiling` page on sentry.io stays empty

![Screenshot 2023-01-03 at 23 58 56](https://user-images.githubusercontent.com/153197/210455278-fa67501d-bdf5-4fbd-85ed-a62cf7b74aba.png)

New transactions do appear but without profiling data. For example this one:

![Screenshot 2023-01-04 at 00 03 19](https://user-images.githubusercontent.com/153197/210455735-b808c8bd-4495-49a8-8f69-29c5ca2f47ef.png)

**Comment by JonasBa** [comment]
@rchl sometimes it may take a bit longer for profiles to be ingested as they have a different ingestion path. Did you end up eventually seeing the profiles or did they just never show up? 

If possible, you can add `debug: true` to the SDK init and it should display some helpful logs there

**Comment by rchl** [comment]
The profiling page is still empty when looking at period of 30 days.

As far `debug` option, I am using it, check my initial comment again. :)

**Comment by JonasBa** [comment]
>As far debug option, I am using it, check my initial comment again. :)
Yeah, sorry about that, I fixed my comment asap as I posted it, but it probably didnt get picked up in the notification.

What I think is happening is that the code you are running between startTransaction and transaction.finish may be very fast to execute and so we only get one single stack sample which is not information to construct a timeline (I added a debug log [here](https://github.com/getsentry/profiling-node/commit/cb6f8aca8a6c447c30b383d66f3f33ca92d53377), but it is not released yet).

If you add some simple await wait(n) where n > 10ms (just to test obv), I would expect the profiles to be shown in the dashboard

**Comment by rchl** [comment]
I've hacked that extra debug code into my installed dependency and I see this:

```
Sentry Logger [log]: [Profiling] Preparing envelope and sending a profiling event.
[Profiling] Invalid traceId: bdad1fd3f1294b7998f047eb50c08bef on profiled event
```

Not sure why is trace ID not UUID4?

**Comment by JonasBa** [comment]
Ooh, wait. What integrations are you using? I think I've seen this happen before, I bet it has something to do with that axios call there (idk if we have auto instrumentation there and could be overriding this).

**Comment by rchl** [comment]
My whole setup is included in the initial comment. You can just save it into a js file and run as long as needed dependencies are installed.

That said, after adding the setTimeout wrapper, the profiling events have finally appeared. Even despite the log about wrong uuid.

**Comment by rchl** [comment]
Actually my example code from the initial comment **does** result in profile being shown in the dashboard currently even though this was not the case when I reported the issue initially. It also generates the warning about wrong trace ID but that doesn't stop it from being shown.

Would it be fair to assume that something has changed on the sentry.io side? I'm sure that nothing has changed in my setup since I've tested it the previous time.

**Comment by rchl** [comment]
Based on my last comment, I think we can close it now.
Hopefully it was due to a fix on the cloud-side as I couldn't explain it otherwise. :)
